CHANGES.md

```diff
@@ -20,6 +22,8 @@ To be released.
 
 ### Added APIs
 
+ -  Added `IValidatorSupportStateDelta` interface.  [[#2733]]
```

Comment:
If our main motivation is hiding validator-related features on `IAccountStateDelta` & `IAccountStateView` against applications, how about hiding this interface too?

Reply:
Applied and squashed!